### PR TITLE
feat: Project resource policy page

### PR DESCRIPTION
**Pull request by agatha197**
follows #2357 
Add Project Resource Policy Page

### TL;DR
This pull request introduces changes related to FormItemWithUnlimited and ProjectResourcePolicyList components.

### What changed?
The changes include:
- Addition of FormItemWithUnlimitedProps in `FormItemWithUnlimited` component
- Updates to `ProjectResourcePolicyList` component with enhanced functionality and UI improvements

### Checklist
- [ ] List Page: name, max folder count, max quota scope size, id, created at
  - [ ] Each item will be shown/hidden according to the server version. 
  - [ ] In particular, it is stated in the schema that `max_vfolder_count` is supported from version 24.03.1, but this is incorrectly documented, and it follows https://github.com/lablup/backend.ai/pull/1993.
- [ ] Create/Modal: Create or modify project resource policy.
  - [ ] Version compatibility is the same as the list.
  - [ ] Unlimited value of `Max Folder Count` is 0.
  - [ ] Unlimited value of `Max Folder Size` is -1.

### Screenshots
<table>
<tr>
<td></td>
 <td>24.03.0b1 (10.82.230.101)</td>
 <td>24.09.0dev1 (main)</td>
</tr>
<tr>
<td>List</td>
<td>

</td>
<td>

</td>
</tr>
<tr>
<td>Create Modal</td>
<td>

</td>
<td>

</td>
</tr>
<tr>
<td>Modify Modal</td>
<td>

</td>
<td>

</td>
</tr>
<tr>
<td>Table Setting</td>
<td>

</td>
<td>

</td>
</tr>
</table>

---

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [x] Minium required manager version
- [x] Specific setting for review (eg., KB link, endpoint or how to setup)
- [x] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by agatha197** [comment]
* **#2423** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2423?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by yomybaby — APPROVED** [review]
LGTM

**Comment by yomybaby** [comment]
### Merge activity

* **May 27, 9:23 PM EDT**: @yomybaby started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2423).
